packages/contracts-bedrock/test/kontrol/deployment/DeploymentSummary.t.sol

```diff
@@ -73,3 +76,42 @@ contract DeploymentSummary_Test is DeploymentSummary, OptimismPortal_Test {
     ///         the L2OutputOracle, which is needed in this test
     function test_isOutputFinalized_succeeds() external override { }
 }
+
+contract DeploymentSummary_TestL1CrossDomainMessenger is DeploymentSummary, L1CrossDomainMessenger_Test {
+    /// @notice super.setUp is not called on purpose
+    function setUp() public override {
+        // Recreate Deployment Summary state changes
+        DeploymentSummary deploymentSummary = new DeploymentSummary();
+        deploymentSummary.recreateDeployment();
+
+        // Set summary addresses
+        optimismPortal = OptimismPortal(payable(OptimismPortalProxyAddress));
+        superchainConfig = SuperchainConfig(SuperchainConfigProxyAddress);
+        l2OutputOracle = L2OutputOracle(L2OutputOracleProxyAddress);
+        systemConfig = SystemConfig(SystemConfigProxyAddress);
+        l1CrossDomainMessenger = L1CrossDomainMessenger(L1CrossDomainMessengerProxyAddress);
```

Comment:
nit, can we make sure all variable names start with a lowercase letter? It seems the vars in the autogenerated `DeploymentSummary.sol` start with capitals

Reply:
The naming of those variables comes from the automatically generated `deployments/hardhat/.deploy`, which stores the name of the contracts (hence the upper case). We could add some extra logic to the json-manipulating scripts to turn them into lowercase, but if it's not too bad we could leave that out of this PR

Reply:
that'd be like a one line fix right? It'd be nice not to accumulate too much stuff we hope to get around to later.

Reply:
Yep! Apologies for the reluctance. It should be fixed with the last commit